nss-day-cohort-51/nss-day-cohort-51.github.io
# Mw fix AddedAlienFacesToCTASection NewDevCardDesign RemovedTechStackURLs swapped tech stack button bg colors

#46 #75 
# Description
Closing Issue 46 -- the Dev Cards are styled based on the latest wireframe.
Closing Issue 75 -- the Tech Stack items have had links removed and hover effects removed. The blocks have been reordered
The Dev Card has had the html and CSS edited to help make the code easier to follow if our cohort website code is looked at by website visitors; made the code more semantic; added BEM format to any new classes; rearranged the order of the action buttons to match the Cohort 51 Website Survey result answers 

## Type of change


# Testing Instructions
- Pull down branch and run in browser
- Check the new DevCard design matches @jmlashen Jake's new DevCard wireframe in Figma
- In the Dev Cards, verify the hover effects of the action buttons (Resume,  works and shows background as blue or mustard based on the Developer type; verify that the social media icons have a hover effect (I added this for a little razzle  dazzle
- Verify that the 3 alien face show on the CTA section; check that the CTA inner three sections stack vertically when the viewport is at a mobile width. 

# Checklist:
